src/main/java/org/mamute/controllers/QuestionController.java

```diff
@@ -166,7 +170,10 @@ public void showQuestion(@Load Question question, String sluggedTitle){
 
 	@Post
 	@CustomBrutauthRules({LoggedRule.class, InputRule.class})
-	public void newQuestion(String title, MarkedText description, String tagNames, boolean watching) {
+	public void newQuestion(String title, MarkedText description, String tagNames, boolean watching,
+							List<Long> attachmentsIds) {
+		List<Attachment> attachments = this.attachments.load(attachmentsIds);
+		attachmentsValidator.validate(attachments);
```

Comment:
Forgot to validate them when adding to existent questions, new answers and existent answers.
Maybe we should do this in a way we don't need to remember